### DISCUSSION TITLE
3D Printed case for PCB_V2B

### DISCUSSION THREAD

**Discussion by jires** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#723] [General]
First of all, @visualapproach, thanks for great job!!

I have made a case to fit PCB_V2B, can be found here: [https://www.printables.com/model/907885-enclosure-wifi-module-for-bestway-lay-z-spa-hottub](url)

In mine design is ESP mounted directly on board, but case should be heigh enough to boards mounted on headers. 

Now, I think manual can be updated that case for latest and greatest pcb exists :)

**Comment by visualapproach** [comment]
Oh, nice! I'll check it out and add it!
